app/helpers/filter_helper.rb

```diff
@@ -12,48 +12,29 @@ def add_filter(query, filter, term)
     if new_query[filter].present? && filter != :sourceFilter
       new_query[filter] << term
       new_query[filter].uniq!
+
+    # Currently, the only single-valued filter provided by TIMDEX API is literaryFormFilter. This filter is not used in
+    # any TIMDEX UI applications. Nevertheless.
+    elsif filter == :literaryFormFilter
+      new_query[filter] = term
     else
       new_query[filter] = [term]
     end
 
-    new_query
-  end
-
-  def nice_labels
-    {
-      accessToFilesFilter: ENV.fetch('FILTER_ACCESS_TO_FILES', 'Access to files'),
-      contentTypeFilter: ENV.fetch('FILTER_CONTENT_TYPE', 'Content type'),
-      contributorsFilter: ENV.fetch('FILTER_CONTRIBUTOR', 'Contributor'),
-      formatFilter: ENV.fetch('FILTER_FORMAT', 'Format'),
-      languagesFilter: ENV.fetch('FILTER_LANGUAGE', 'Language'),
-      literaryFormFilter: ENV.fetch('FILTER_LITERARY_FORM', 'Literary form'),
-      placesFilter: ENV.fetch('FILTER_PLACE', 'Place'),
-      sourceFilter: ENV.fetch('FILTER_SOURCE', 'Source'),
-      subjectsFilter: ENV.fetch('FILTER_SUBJECT', 'Subject')
-    }
-  end
-
-  def gdt_sources(value, category)
-    return value if category != :sourceFilter
-
-    return 'Non-MIT institutions' if value == 'opengeometadata gis resources'
-
-    return 'MIT' if value == 'mit gis resources'
-
-    value
```

Comment:
Note to reviewer: this is the result of reordering the methods in the filter helper, as noted in the side effects. (See also lines 70-123.)